theseus/src/api/profile.rs

```diff
@@ -234,6 +234,26 @@ pub async fn run_credentials(
 
     let env_args = &settings.custom_env_args;
 
+    // Post post exit hooks
+    let post_exit_hooks =
+        &profile.hooks.as_ref().unwrap_or(&settings.hooks).post_exit;
+
+    let post_exit_hooks: Vec<Command> = post_exit_hooks
+        .iter()
+        .filter_map(|hook| {
+            let mut cmd = hook.split(' ');
+            if let Some(command) = cmd.next() {
+                let mut command = Command::new(command);
+                command
+                    .args(&cmd.map(|s| s.to_string()).collect::<Vec<String>>())
+                    .current_dir(path);
+                Some(command)
+            } else {
+                None
+            }
+        })
+        .collect();
```

Comment:
weren't we only doing one command for the post exit hooks?

Reply:
Yeah- it is only one command .but in this part its stored as a vec because previously it went from an iter to a vec (and you can iter over an Option so it worked mostly in place without changing this aspect)

I agree this is misleading though- so fixed.